editor.planx.uk/src/@planx/components/Pay/Editor.test.tsx

```diff
@@ -0,0 +1,320 @@
+import { User } from "@opensystemslab/planx-core/types";
+import { fireEvent, waitFor } from "@testing-library/react";
+import { FullStore, vanillaStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+import { DndProvider } from "react-dnd";
+import { HTML5Backend } from "react-dnd-html5-backend";
+import { act } from "react-dom/test-utils";
+import { axe, setup } from "testUtils";
+
+import PayComponent from "./Editor";
+
+describe("Pay component - Editor Modal", () => {
+  it("renders", () => {
+    const { getByText } = setup(
+      <DndProvider backend={HTML5Backend}>
+        <PayComponent id="test" />
+      </DndProvider>,
+    );
+    expect(getByText("Payment")).toBeInTheDocument();
+  });
+
+  // Currently failing, Editor not a11y compliant
+  it.skip("should not have any accessibility violations upon initial load", async () => {
+    const { container } = setup(
+      <DndProvider backend={HTML5Backend}>
+        <PayComponent id="test" />
+      </DndProvider>,
+    );
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
+
+  describe("GOV.UK Pay Metadata section", () => {
+    jest.setTimeout(20000);
```

Comment:
Spent way too long trying to come up with something better than this!

As these are some of the only Editor tests I'm hitting up against a few things here - I think it's mainly the "update not wrapped in an act()" warning which we've butted up against on other occasions (https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning)

I'd like to come back and take another look at this, but in order to progress this PR I'd sidestepped the issue for now by just allowing the tests to run for longer.